vsts.yml

```diff
@@ -1,3 +1,22 @@
+# The config expects the following environment variables to be set:
+#  - "Custom.UseArtifacts.BuildId" aka "CUSTOM_USEARTIFACTS_BUILDID"
+#      Set it to a build number, e.g. 15912, to download artifacts
+#      from that build and only run the tests, instead of building from scratch.
+#  - "ELECTRON_RELEASE" Set it to '1' upload binaries on success.
+#  - "GN_CONFIG" Full path to a build config, e.g. '//electron/build/args/testing.gn'.
+#  - "GN_EXTRA_ARGS" Additional gn args, e.g. 'is_mas_build=true'.
+#  - "NOTIFY_SLACK" Set it to '1' to enable Slack notifications.
+#  - "RUN_TESTS" Set it to '1' to run Electron's tests.
+#  - "SLACK_WEBHOOK" Slack hook URL to send notifications.
```

Comment:
This variable is `slack_webhook` and is a private variable meaning you can't access it as an environment variable, so I'm not sure we need to document here.

Reply:
Removed